### PR TITLE
Add changes for `edge-19.8.3`

### PR DESCRIPTION
**Pull request by siggy**
Signed-off-by: Andrew Seigner <siggy@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+    annotations
+  * Fixed `linkerd stat -h` example commands (thanks @ethan-daocloud!)
+* Proxy
+  * Added `request_handle_us` histogram to measure proxy overhead
+  * Updated Tap server to require identity
+  * Formatted log records consistently
+  * Fixed tap response streams never ending
+  * Fixed Destination service resource leak by hanging up destination
+    resolutions when dropped
+  * Authority label reflects logical dst
```
Comment (olix0r) [inline CHANGES.md]: I would remove this proxy message, as well as those below.

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+`Kubernetes cluster monitoring` Grafana dashboard.
+
+* CLI
+  * Added TrafficSplit metrics via `linkerd stat ts` subcommand
+  * Fixed `linkerd uninject` not removing `linkerd.io/inject: enabled`
+    annotations
+  * Fixed `linkerd stat -h` example commands (thanks @ethan-daocloud!)
+* Proxy
+  * Added `request_handle_us` histogram to measure proxy overhead
+  * Updated Tap server to require identity
```
Comment (olix0r) [inline CHANGES.md]: Updated the Tap server to only admit requests from the control plane's Tap controller

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+  * Formatted log records consistently
```
Comment (olix0r) [inline CHANGES.md]: omit

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+  * Fixed tap response streams never ending
```
Comment (olix0r) [inline CHANGES.md]: Fixed a bug that could prevent taps from completing

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+  * Fixed Destination service resource leak by hanging up destination
+    resolutions when dropped
```
Comment (olix0r) [inline CHANGES.md]: Fixed a bug that could cause the proxy to leak service discovery resolutions to the Destination service

Comment (olix0r) [reply CHANGES.md]: better: Destination _controller_

File: CHANGES.md (hunk 2 above)
Comment (adleong) [inline CHANGES.md]: "Updated Tap server to only accept tap requests from the tap controller" would be a more functional description of the change

File: CHANGES.md (hunk 4 above)
Comment (adleong) [inline CHANGES.md]: Fixed a bug where tap would stop streaming after a short amount of time

File: CHANGES.md (hunk 1 above)
Comment (adleong) [inline CHANGES.md]: This should be in the past tense.

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,39 @@
...
+  * Authority label reflects logical dst
+  * Replaced futures_watch with tokio::sync::watch
```
Comment (adleong) [inline CHANGES.md]: Not sure if any of these next 4 bullet points are user-facing enough to mention.

File: CHANGES.md [hunk 7]
```diff
@@ -1,3 +1,39 @@
...
+* Web UI
+  * Added `Kubernetes cluster monitoring` Grafana dashboard with cluster and
+    containers metrics
+* Internal
+  * Updated `linkerd install` and `linkerd upgrade` to use Helm charts for
+    templating
+  * Pinned Helm tooling to `v2.14.3`
+  * Added Helm integration tests
+  * Added container CPU and memory usage to `linkerd-heartbeat` requests
+  * Removed Tap from the Public API, now only available via authenticated TAP
```
Comment (adleong) [inline CHANGES.md]: Should this go into a `Controller` section?

File: CHANGES.md [hunk 8]
```diff
@@ -1,3 +1,39 @@
+## edge-19.8.3
+
+This edge release introduces a new `linkerd stat ts` subcommand, enabling
+retrieval of TrafficSplit metrics. It also introduces a
```
Comment (wmorgan) [inline CHANGES.md]: s/enabling retrieval of TrafficSplit metrics/to show traffic split metrics/.

File: CHANGES.md [hunk 9]
```diff
@@ -1,3 +1,39 @@
...
+`Kubernetes cluster monitoring` Grafana dashboard.
```
Comment (wmorgan) [inline CHANGES.md]: Feels like more of a " than a `

File: CHANGES.md [hunk 10]
```diff
@@ -1,3 +1,39 @@
...
+
+* CLI
+  * Added TrafficSplit metrics via `linkerd stat ts` subcommand
```
[the remark this answers is not in the document]
Comment (wmorgan) [reply CHANGES.md]: I think we should, and let's call it "traffic split" instead of TrafficSplit here too please

File: CHANGES.md (hunk 6 above)
Comment (wmorgan) [reply CHANGES.md]: Roll 'em up

File: CHANGES.md [hunk 11]
```diff
@@ -1,3 +1,35 @@
...
+show traffic split metrics. It also introduces a "Kubernetes cluster monitoring"
+Grafana dashboard.
+
+* CLI
+  * Added traffic split metrics via `linkerd stat trafficsplits` subcommand
+  * Fixed `linkerd uninject` not removing `linkerd.io/inject: enabled`
+    annotations
+  * Fixed `linkerd stat -h` example commands (thanks @ethan-daocloud!)
+* Controller
+  * Removed Tap from the Public API, now only available via authenticated TAP
```
Comment (wmorgan) [inline CHANGES.md]: How about: Removed old non-authenticated endpoints for `linkerd tap`, which is now only available via the authenticated API?

File: CHANGES.md [hunk 12]
```diff
@@ -1,3 +1,35 @@
...
+    APIServer
+* Proxy
+  * Added `request_handle_us` histogram to measure proxy overhead
+  * Updated the Tap server to only admit requests from the control plane's Tap
```
Comment (wmorgan) [inline CHANGES.md]: How are we capitalizing tap? I see three different versions in these notes

File: CHANGES.md (hunk 11 above)
Comment (siggy) [reply CHANGES.md]: Actually both that wording and your suggestion are not quite true. They will be in stable-2.5. For now changing this to:
`Removed unauthenticated Tap from the Public API`

Comment (wmorgan) [reply CHANGES.md]: Just... make the capitalization make me happy

**Pull request merged**